docs/introduction/get-started/p2p-public.md

```diff
@@ -30,10 +30,10 @@ To do this, we use two terminals with their own state (using their own `miden-cl
 
     Accept the defaults for `Protocol`, `Node RPC Port`, and `Sqlite file path`by clicking enter.
 
-    When `Host` comes up, enter the IP that the Miden team supplies:
+    When `Host` comes up, enter `18.203.155.106`
 
     ```sh
-    Host (default: localhost): xxx.xxx.xxx.xxx
+    Host (default: localhost): 18.203.155.106
```

Comment:
Are you sure this won't change?

Reply:
For the next 2 weeks, I think its fine. Whatever makes the process for the users easier.

Reply:
it's just that the docs go out of date really quickly when you're specific like this ...

Reply:
I promise we will change that, but for now I don't see a better solution. We will add the IP to the client soon @igamigo